src/modules/cmdpal/Microsoft.CmdPal.UI/Controls/IconBox.cs

```diff
@@ -98,10 +98,7 @@ private static void OnSourceKeyPropertyChanged(DependencyObject d, DependencyPro
                     {
                         await @this.SourceRequested.InvokeAsync(@this, eventArgs);
 
-                        if (eventArgs.Value != null)
-                        {
-                            @this.Source = eventArgs.Value;
-                        }
+                        @this.Source = eventArgs.Value;
```

Comment:
Ah, thanks, I know we discussed this last week, and I forgot to go back and fix it. 😅

Reply:
Future note for if we pull this in Toolkit later, is we could have a `FallbackSource` property to display when the key lookup returns null here. I don't think we have a scenario for this currently, but just a thought.